src/components/Employee/Deductions/DeductionsForm/DeductionsFormV2.tsx

```diff
@@ -0,0 +1,162 @@
+import { useTranslation } from 'react-i18next'
+import { useState } from 'react'
+import { useGarnishmentsListSuspense } from '@gusto/embedded-api/react-query/garnishmentsList'
+import { useGarnishmentsGetChildSupportDataSuspense } from '@gusto/embedded-api/react-query/garnishmentsGetChildSupportData'
+import styles from './DeductionsForm.module.scss'
+import ChildSupportForm from './ChildSupportForm'
+import CustomDeductionForm from './CustomDeductionForm'
+import {
+  BaseComponent,
+  type BaseComponentInterface,
+  type CommonComponentInterface,
+  useBase,
+} from '@/components/Base'
+import { Grid } from '@/components/Common/Grid/Grid'
+import { useI18n } from '@/i18n'
+import { componentEvents } from '@/shared/constants'
+import { useComponentContext } from '@/contexts/ComponentAdapter/useComponentContext'
+import { useComponentDictionary } from '@/i18n/I18n'
+import CaretLeft from '@/assets/icons/caret-left.svg?react'
+interface DeductionsFormProps extends CommonComponentInterface<'Employee.Deductions'> {
+  employeeId: string
+  deductionId?: string | null
+}
+
+export function DeductionsFormV2(props: DeductionsFormProps & BaseComponentInterface) {
```

Comment:
This is just a preference, but cam we not call it V2? Speakeasy/OpenAPI already do this a bunch and it drives me nuts. 

Are we planning on getting rid of the older v1?